yellowbrick/text/postag.py

```diff
@@ -157,10 +155,11 @@ def fit(self, X, y=None, **kwargs):
         self : instance
             Returns the instance of the transformer/visualizer
         """
-        if self.stack and y is None:
-            raise YellowbrickValueError("Specify y for stack=True")
-        
-        self.labels_=np.unique(y)
+        self.labels_=['documents']
```

Comment:
Ok, this makes sense to me - but we should make a note of it in the docstring when we describe `self.labels_` - nice rewrite for the exception handling section below!